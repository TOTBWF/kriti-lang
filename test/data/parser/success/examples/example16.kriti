{{ if $[0].isActive == false }} true {{ else }} false {{ end }}