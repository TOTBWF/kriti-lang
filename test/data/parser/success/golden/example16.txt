Iff
    ( SourcePosition
        { _sourceName = "sourceName"
        , _line = 1
        , _column = 1
        }
    , Just
        ( SourcePosition
            { _sourceName = "sourceName"
            , _line = 1
            , _column = 21
            }
        )
    )
    ( Eq
        ( SourcePosition
            { _sourceName = "sourceName"
            , _line = 1
            , _column = 21
            }
        , Just
            ( SourcePosition
                { _sourceName = "sourceName"
                , _line = 1
                , _column = 22
                }
            )
        )
        ( Path
            [
                (
                    ( SourcePosition
                        { _sourceName = "sourceName"
                        , _line = 1
                        , _column = 7
                        }
                    , Just
                        ( SourcePosition
                            { _sourceName = "sourceName"
                            , _line = 1
                            , _column = 8
                            }
                        )
                    )
                , Obj "$"
                )
            ,
                (
                    ( SourcePosition
                        { _sourceName = "sourceName"
                        , _line = 1
                        , _column = 8
                        }
                    , Just
                        ( SourcePosition
                            { _sourceName = "sourceName"
                            , _line = 1
                            , _column = 10
                            }
                        )
                    )
                , Arr 0
                )
            ,
                (
                    ( SourcePosition
                        { _sourceName = "sourceName"
                        , _line = 1
                        , _column = 11
                        }
                    , Just
                        ( SourcePosition
                            { _sourceName = "sourceName"
                            , _line = 1
                            , _column = 19
                            }
                        )
                    )
                , Obj "isActive"
                )
            ]
        ) ( Boolean False )
    ) ( Boolean True ) ( Boolean False )